From 0f826001ab89461aff6ac536a375f521386c0d02 Mon Sep 17 00:00:00 2001
From: Kazunori Kobayashi <kkobayas@igel.co.jp>
Date: Thu, 31 May 2012 10:33:34 +0900
Subject: [PATCH 25/97] ext/dfbvideosink: eliminate a warning message

The variable "line" in gst_dfbvideosink_show_frame() is only used when
HAVE_SHVIO is not defined.
---
 ext/directfb/dfbvideosink.c |    4 +++-
 1 file changed, 3 insertions(+), 1 deletion(-)

diff --git a/ext/directfb/dfbvideosink.c b/ext/directfb/dfbvideosink.c
index 3afa575..57266b5 100644
--- a/ext/directfb/dfbvideosink.c
+++ b/ext/directfb/dfbvideosink.c
@@ -1760,12 +1760,14 @@ gst_dfbvideosink_show_frame (GstBaseSink * bsink, GstBuffer * buf)
   if (mem_cpy) {
     IDirectFBSurface *dest = NULL, *surface = NULL;
     guint8 *data;
-    gint dest_pitch, src_pitch, line;
+    gint dest_pitch, src_pitch;
     GstStructure *structure;
 #if defined(HAVE_SHVIO)
     DFBSurfacePixelFormat src_format, dst_format;
     struct ren_vid_surface viosurface[2];
     int ret;
+#else
+    gint line;
 #endif
 
     /* As we are not blitting no acceleration is possible. If the surface is
-- 
1.7.10.4

